frontend/src/assets/stylesheets/components/post.sass

```diff
@@ -97,6 +106,13 @@
     svg
       margin-right: 2px
```

Comment:
Let's refactor that, if the icon is prepared well it shouldn't have margin-right like that